naga/src/back/wgsl/writer.rs

```diff
@@ -627,6 +627,12 @@ impl<W: Write> Writer<W> {
                 for handle in range.clone() {
                     let info = &func_ctx.info[handle];
                     let expr_name = if let Some(name) = func_ctx.named_expressions.get(&handle) {
+                        if name.starts_with("_naga_phony") {
```

Comment:
What happens if we don't special-case this?

Reply:
it gets emitted as `let` instead of `_`

Reply:
this is now removed